types/module/module.go

```diff
@@ -95,15 +96,15 @@ type HasGRPCGateway interface {
 // HasGenesis is the extension interface for stateful genesis methods.
 type HasGenesis interface {
```

Comment:
Can we delete this and alias the appmodule v2 api in appmodule v1?

Reply:
checking